src/components/FileUploader/FileUploader.js

```diff
@@ -112,25 +107,14 @@ export class FileUploaderButton extends Component {
   }
 }
 
-export class Filename extends Component {
-  static propTypes = {
-    style: PropTypes.object,
-    status: PropTypes.oneOf(['edit', 'complete', 'uploading']),
-    tabIndex: PropTypes.number,
-    onKeyDown: PropTypes.func,
-  };
-
-  static defaultProps = {
-    onKeyDown: () => {},
-    status: 'uploading',
-    style: {},
-    tabIndex: 0,
-  };
-
-  render() {
-    const { iconDescription, status, style, ...other } = this.props;
-
-    if (status === 'uploading') {
+export function Filename({
```

Comment:
We cannot make a breaking change tp `<Filename>` either - Can we make it non-breaking, or do you want to do `<FilenameV2>`?

Reply:
This change should be a non-breaking change. I am converting the class component to a stateless function. Have I mistakenly removed a feature that should be present?